backend/app/models.py

```diff
@@ -35,6 +44,20 @@ class GerryDBTable(TimeStampMixin, SQLModel, table=True):
     name: str = Field(nullable=False, unique=True)
 
 
+# Since `table_uuid` foreign key is unique, is there a good reason for tiles to be a separate table?
+class GerryDBTiles(TimeStampMixin, SQLModel, table=True):
+    uuid: str = Field(sa_column=Column(UUIDType, unique=True, primary_key=True))
+    table_uuid: str = Field(
+        sa_column=Column(UUIDType, ForeignKey(GerryDBTable.uuid), unique=True)
+    )
+    s3_path: str = Field(nullable=False)
```

Comment:
(q) what's the difference between this s3 path and the tiles_s3 path in the GerryDBViewPublic class?

Reply:
IMO shouldn't store it twice to make consistency easier, and if we're finding ourselves always having to join both tables at the same granularity, makes sense to make it one from the get-go.

Reply:
combined these two tables!